pages/lessons_incubation.md

```diff
@@ -0,0 +1,235 @@
+    require.) This is not only a pragmatic requirement, but also
+    confirms interest in the most concrete way.
+
+5.  No equivalent lessons exist—at least, none that do not assume
+    prior knowledge, are living documents updated by their community,
+    and use a hands-on teaching style.
+
+6.  The lesson materials will be freely available under a license that
+    requires no more than attribution, such as the Creative Commons -
+    Attribution license (CC-BY). We (strongly) prefer that lessons use
+    open source tools, but do not require that. Similarly, we do not
+    require that lessons be created from scratch, and in fact prefer
+    contributors to recycle materials they have tested in the
+    classroom so long as they meet licensing stipulations. We do
+    require that lessons use our [templates]({{site.github_io_url}}/lesson-example/)
+    and [teaching style]({{site.github_io_url}}/instructor-training/).
+
+Data Carpentry and Software Carpentry’s Steering Committees may make
+exceptions to these requirements in special cases, but such exceptions
+will be rare.
+
+## 2. What New Lesson Proposals Need
+
+As per point 4 above, proposals for new lessons or workshops must be
+backed by concrete statements of support for initial development,
+delivery, and maintenance. Such support should include:
+
+1.  One or two lead developer(s) who have made a credible commitment
+    to develop and deliver the first complete version of the
+    lesson. (Based on our experience with Data Carpentry’s lessons on
+    genomics and geospatial data, a new two-day workshop requires 0.5
+    person-years of effort.) The actual effort may be less when
+    existing material can be recycled, or when the lesson lead has
+    significant prior experience with lesson development.
+
+2.  At least two trial sites, where early adopters with both domain
+    expertise and experience teaching our general workshops have
+    committed to delivering the first complete version of the lesson
+    and providing detailed feedback.
+
+3.  Administrative support so that Data and Software Carpentry can
+    coordinate work, assess outcomes, and report on results. Based on
+    experience with developing Data Carpentry, we budget this at
+    US$10,000 for a workshop US$5,000 for a half-day lesson. This cost
+    may be incorporated into a partnership agreement with the
+    sponsoring organization.
+
+4.  If a full two-day workshop is being developed, the budget must
+    also include travel money for a hackathon at which early adopters
+    (both those at trial sites and others) can work together to refine
+    the first version based on early experience teaching it. This is
+    usually a 3-day event for 10–15 people whose costs to participate
+    are covered, at which the lead developer(s) will usually teach the
+    entire material for early adopters to observe. Based on Data
+    Carpentry’s experience, such a hackathon costs roughly US$20,000.
```

Comment:
This seems like a big ask ;).  Have any existing SWC lessons had a three-day, $20k hackathon with 10-15 people meeting in person to work on the lesson?